### PR TITLE
Communicate with bootnodes, mailservers and whisper nodes by :443 (staging, test, beta)

### LINKED ISSUE #6351: Chat non-functional on networks that block ports other than 80/443
Visited the Liberty Science Center today and to my surprise found a pic of Vitalik on the wall - turns out he made it to their list of resident Genuises: https://lsc.org/index.php?cID=1592 :)

They also have a free wifi that, like many other free wifi's on airports, cafes and libraries that I've been to and tried using Status, blocks anything other than web traffic, usually by port filtering (helps them avoid most torrenting for example)

Whisper, with its default settings, runs a devp2p overlay network on an odd port in the 30k range, and so do all our servers making the chat unreachable on such networks (status app shows something like "connecting to peers" indefinitely).

Possible short term fix: run a few whisper nodes on said ports (won't help with smarter, content-based packet filters, but these are less common)

<blockquote><div> Liberty Science Center</div><div><strong><a href="https://lsc.org/support-us/genius-gala-draft/genius-gallery">Genius Gallery</a></strong></div><div>View the full list of winners from our annual Genius Gala event.</div></blockquote>

### LINKED ISSUE #6645: Status could not connect to peers on public wifi network
### Description

*Type*: Bug

*Summary*: 

I was connected to a public wifi that had internet. All pages like google.com loaded fine. But, Status kept showing "Connecting to peers..." and did not fetch any messages. I tried on iOS, Android and Desktop. It would be great to investigate it as it may be the case for many of our users. I was looking at the logcat, but there was nothing that might be related. I'm attaching geth.log to investigate it.

[geth.log](https://github.com/status-im/status-react/files/2560978/geth.log)

### Reproduction

No exact steps to reproduce. I was on open public wifi network.

### Additional Information
* Status version: nightly, Nov 8
* Operating System: Android, iOS, Desktop

### PR DESCRIPTION
**Pull request by mandrigin**
fixes #6645
fixes #6351

### Summary:

Essentially, we use HTTPS port :443 to connect to our nodes.

TODO:
- [x] Test, then deploy :443 mirroring on the `eth.beta` too.

#### Areas that maybe impacted (optional)
**Functional**
- 1-1 chats
- public chats
- group chats
- mailservers

**NB!** "test" or "staging" fleet should be used to test it.

### Steps to test:
- Restrict connections on the network to ports :443 and :80
- Launch Status, switch to the staging fleet.
- Chat should work, the app shouldn't be offline.

status: ready

### CODE REVIEW

**Comment by mandrigin** [comment]
I tried it locally in iOS simulator where Little Snitch blocked all Status trafic except :80 and :443 and it seems to help. On `eth.beta` the app is ever-"connecting" or "Offline" whereas on `eth.staging` I can connect and send message.

**Comment by mandrigin** [comment]
@jakubgs it looks like we can proceed and setup the same stuff on `eth.beta`. But keep in mind public keys of the nodes, make sure that all of them will be still available on `eth.beta`.

This PR changes a few of them for `eth.staging` I don't exactly know why.

**Comment by jakubgs** [comment]
>This PR changes a few of them for eth.staging I don't exactly know why.

Ah yeah, because I was updating keys in Consul by hand, I must have missed one or two, my bad. I'll make sure to not do that on Beta.

**Comment by mandrigin** [comment]
@jakubgs so, ping me when you will do that and then I will be able to update and un-WIP this PR.

File: resources/config/fleets.json [hunk 1]
```diff
@@ -50,18 +53,19 @@
...
+                "boot-01.gc-us-central1-a.eth.staging": "enode://10a78c17929a7019ef4aa2249d7302f76ae8a06f40b2dc88b7b31ebff4a623fbb44b4a627acba296c1ced3775d91fbe18463c15097a6a36fdb2c804ff3fc5b35@35.238.97.234:443"
             },
             "mail": {
-                "mail-01.ac-cn-hongkong-c.eth.staging": "enode://b74859176c9751d314aeeffc26ec9f866a412752e7ddec91b19018a18e7cca8d637cfe2cedcb972f8eb64d816fbd5b4e89c7e8c7fd7df8a1329fa43db80b0bfe@47.52.90.156:30504",
-                "mail-01.do-ams3.eth.staging": "enode://69f72baa7f1722d111a8c9c68c39a31430e9d567695f6108f31ccb6cd8f0adff4991e7fdca8fa770e75bc8a511a87d24690cbc80e008175f40c157d6f6788d48@206.189.240.16:30504",
-                "mail-01.gc-us-central1-a.eth.staging": "enode://e4fc10c1f65c8aed83ac26bc1bfb21a45cc1a8550a58077c8d2de2a0e0cd18e40fd40f7e6f7d02dc6cd06982b014ce88d6e468725ffe2c138e958788d0002a7f@35.239.193.41:30504"
+                "mail-01.ac-cn-hongkong-c.eth.staging": "enode://9ea7fbf9bc3844b7f154464c39935d0af71f6d4deb312fbd67b95785389f68dda221276512b8186e71453ef95148cf3ccd5ecc05a2e7deb64182d8ca36916746@47.52.90.156:443",
```
Comment (jakubgs) [inline resources/config/fleets.json]: These have been updated in https://fleets.status.im/ too, they shouldn't change here.

Comment (mandrigin) [reply resources/config/fleets.json]: thanks!

Comment (mandrigin) [reply resources/config/fleets.json]: fixed

**Review by pedropombeiro — APPROVED** [review]

**Review by jakubgs — APPROVED** [review]

**Comment by Serhy** [comment]
Latest builds here for Android, iOS, MacOS builds were tested. 
I can't tweak with coworking wifi settings, so MacOS (and iOS with Android too) tested on a matter of regressions which were not found.
I also went to macdonalds and local store nearby to use their public wifi (with iOS and Android devices) :) Confirmed with each mailserver there, from staging fleet and some mailsevers from fleets.beta (except do-ams3.eth.beta mailservers), that offline inbox history fetched, can exchange messages between two contacts, can receive PNs.

From `netstat` output (for emulator) it's visible that mailservers ips are using https (so 443) port with established connection, means that with using updated ports everything works properly.

@mandrigin do we intentionally keep mail-03.do-ams3.eth.beta, mail-03.do-ams3.eth.beta, mail-03.do-ams3.eth.beta staying to use 30504 port? If yes, then it ready for merge

**Comment by mandrigin** [comment]
@Serhy yeah, I just followed whatever is in http://fleets.status.im

**Comment by jakubgs** [comment]
I don't see `30504` anywhere in https://fleets.status.im, so this change is just a bit stale.

<blockquote></blockquote>

**Comment by mandrigin** [comment]
fixed that

**Comment by mandrigin** [comment]
@Serhy can we re-run e2e tests on this branch when the build is complete? I fixed the last ports, so if e2e pass, I think it is safe to merge.

**Comment by Serhy** [comment]
@mandrigin, @jakubgs thanks for update. 
Once builds made, will run e2e tests, yes.

**Comment by mandrigin** [comment]
@Serhy android-e2e is done, so we can run tests now.

**Comment by Serhy** [comment]
e2e tests are in progress now...

**Comment by Serhy** [comment]
@mandrigin for me `mail-01.do-ams3.eth.beta` always fails to establish the connection.
It's all with nightly build (where 30504 port), but once updated to the latest PR builds the 'Failed to connect to mailserver' pop-up appears.
Only with `mail-01.do-ams3.eth.beta`, the rest mailservers are ok.

**Comment by jakubgs** [comment]
@Serhy what is your IP or peer ID? You might be a stale peer on that mailserver.

**Comment by Serhy** [comment]
@jakubgs My IP: 213.160.136.62

**Comment by jakubgs** [comment]
Hmmm, I don't see you in peers on `mail-01.do-ams3.eth.beta`.

But I checked on the host and it was missing the firewall rule:
```bash
admin@mail-01.do-ams3.eth.beta:~ % sudo iptables -t nat -L PREROUTING
Chain PREROUTING (policy ACCEPT)
target     prot opt source               destination         
DOCKER     all  --  anywhere             anywhere             ADDRTYPE match dst-type LOCAL
```
And I saw that the healtcheck was failing:
https://canary.status.im/icingaweb2/dashboard#!/icingaweb2/monitoring/service/show?host=mail-01.do-ams3.eth.beta&service=statusd-mail
So I ran ansible on it, it should be fine now:
```bash
admin@mail-01.do-ams3.eth.beta:~ % sudo iptables -t nat -L PREROUTING
Chain PREROUTING (policy ACCEPT)
target     prot opt source               destination         
DOCKER     all  --  anywhere             anywhere             ADDRTYPE match dst-type LOCAL
REDIRECT   tcp  --  anywhere             anywhere             tcp dpt:https /* Redirect 443 to 30504 */ redir ports 30504
```
Thanks for reporting this.

**Comment by Serhy** [comment]
> I don't see you in peers on `mail-01.do-ams3.eth.beta`

Yeah, at that time I was on this PR build, and failing to establish the connection with mail-01. 
Thanks for noticing and adding the redirect rule! Yes, I can confirm with MacOS and Android connection established with `mail-01.do-ams3.eth.beta`, offline messages featched. 

iOS builds failing to succeed, @jakubgs could you take a look please?

**Comment by jakubgs** [comment]
Yeah, no idea what that's about. Xcode fails to build but there is no error whatsoever.

**Comment by mandrigin** [comment]
@Serhy @jakubgs  is it just this PR? makes no sense, it fails to compile Firebase...

**Comment by mandrigin** [comment]
@Serhy now it passes. I've no idea what was it...

**Comment by Serhy** [comment]
Cool! Mailservers are OK with iOS build too. 
Lets merge this, @mandrigin !

**Pull request merged**